test-poc/framework/src/main/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigSourceProvider

```diff
@@ -0,0 +1,2 @@
+io.smallrye.config.PropertiesConfigSourceProvider
+io.smallrye.config.DotEnvConfigSourceProvider
```

Comment:
@wojnarfilip please, rebase to latest change and focus on smallrye support of .env files (out of the box), I would propose to remove this class then: test-poc/framework/src/main/java/org/keycloak/test/framework/config/Config.java